cybered/hashing/views.py

```diff
@@ -22,23 +22,31 @@ def scoped(text):
     return HashingConfig.name + ":" + text
 
 
+# TODO: There might be a way to hoist these base classes out of the individual modules, but note
```

Comment:
I don't like forcing every module to copy paste these base classes, but I can't think of a better way to do it. Can you?